src/i18n/locales/en/translation.json

```diff
@@ -717,7 +717,18 @@
       "title": "Receive",
       "actions": {
         "create_invoice": "Create Invoice",
-        "copy_lightning_address": "Copy Lightning Address"
+        "invoice": {
+          "title": "Lightning invoice",
+          "description": "Request instant and specific amount payments"
+        },
+        "bitcoin_address": {
+          "title": "Bitcoin address",
+          "description": "Request instant and specific amount payments"
```

Comment:
this is wrong right? onchain is not instant

Reply:
What about "Receive on-chain payments to your Alby account"?

Reply:
Good catch, that should be a different message here! :sweat_smile: